bgpd/bgp_snmp_bgp4v2.c

```diff
@@ -360,6 +360,8 @@ static uint8_t *bgpv2PeerErrorsTable(struct variable *v, oid name[],
 				msg_str = bgp_notify_admin_message(
 					msg_buf, sizeof(msg_buf),
 					(uint8_t *)notify.data, notify.length);
+				if(!msg_str)
```

Comment:
Why don't we just make bgp_notify_admin_message return an empty string instead of NULL?  Then whomever comes after this will not have an issue at all.

Reply:
impact would be the following:

-bgpd/bgp_vty.c:11178 , a test on NULL return value is made
-bgp_notify_admin_message function

i think that a NULL pointer is more natural in case of failure than an empty string.

Reply:
No your analysis is just plain wrong.  I can't think of any other function in FRR that is effectively a 2str() function that returns NULL.  Nor do I think it's appropriate that we turn the rest of FRR into a series of tests for 2str() functions to ensure that we are not going to print a "(NULL)" in a string.  This prevents you from using it directly in a log message.  Please make this code work the same as every other 2str() function in FRR

Reply:
ok taken into account